### PR TITLE
How I got into tech?

### PR DESCRIPTION
**Pull request by PragatiVerma18**
Contributed an article that explains why and how I got into tech.

### CODE REVIEW

File: my-tech-journey/genesis.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## How I got into tech?
+
+To begin with my writing journey, I feel that I must introduce myself, and what's better than to write about my journey in tech and give an insight to what brought me here.
```
Comment (swapagarwal) [inline my-tech-journey/genesis.md]:
```suggestion
To begin with my writing journey, I feel that I must introduce myself, and what's better than to write about my journey in tech and give an insight into what brought me here.
```

File: my-tech-journey/genesis.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+>"When you grow up, you tend to get told the world is the way that it is, and your life is just to live your life inside the world and try not to bash into the walls too much. But that's a very limited life."
+
+I always wanted to study Computer Science but getting admission in a good college with CS/IT stream is no more an easy deal. You got to almost kill yourself studying Physics, Chemistry and Maths to pass those entrance exams with good marks. I passed the entrance exams I gave but was just on the borderline with the cut-off marks in hand, which led to me a third-tier college affiliated to AKTU, although, I was happy that it was quite famous for it's Computer Science Graduates doing really well with decent placement offers in big firms. Meanwhile, I started writing articles for [fossbytes.com](https://fossbytes.com/author/pragati/).
```
Comment (swapagarwal) [inline my-tech-journey/genesis.md]:
```suggestion
I always wanted to study Computer Science but getting admission in a good college with CS/IT stream is no more an easy deal. You got to almost kill yourself studying Physics, Chemistry and Maths to pass those entrance exams with good marks. I passed the entrance exams I gave but was just on the borderline with the cut-off marks in hand, which led to me a third-tier college affiliated to AKTU, although, I was happy that it was quite famous for its Computer Science Graduates doing really well with decent placement offers in big firms. Meanwhile, I started writing articles for [fossbytes.com](https://fossbytes.com/author/pragati/).
```

File: my-tech-journey/genesis.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+#### Outset of College
+As I bumped into college life, I got introduced to the cool and awesome technical student societies/clubs of the college, and then it become my prime goal to get in the best of them and try to improve my technical skills and get the best out of what I have got. Moreover, I kept reading books about Elon Musk, Linus Torvalds and Bill Gates. In the first year itself, I was introduced to the world of competitive coding, I thought maybe I can do this. So, I began practicing on platforms like [CodeChef](https://www.codechef.com/users/pragativerma18) and [HackerRank](https://www.hackerrank.com/itispragativerma) etc. It went smooth for sometime, I was able to do 2-3 questions in a long challenge but there came a time I realised that this wasn't I was looking for. I began exploring more options, where I thought that web development might be fun, so I switched to development.
```
Comment (swapagarwal) [inline my-tech-journey/genesis.md:12]: Maybe you can expand on some learnings from these books?
How did you realize competitive coding wasn't "it"?

File: my-tech-journey/genesis.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+#### Internship and Hackathons
+Meanwhile, I was exploring the world of open source software and decided to learn how to use git and GitHub, such that I can try to contribute to open source in whichever way possible. I also kept looking for internship opportunities and fortunately, I got a web development intern role at a startup named [Uneako](http://uneako.com/). We also started exploring hackathons nearby our college, our first hackathon being- hackit@BVP1.0 at BVP,Delhi. It was an enriching experience to be part of these hackathons and meetups, as they open a window to the better in you. They give us a chance to introspect what we have been lacking or what more is to be done to be the winner in your chosen arena of work or ambition. Often, they are the inspiration of new beginnings or the genesis of better future.
```
Comment (swapagarwal) [inline my-tech-journey/genesis.md:18]: > They give us a chance to introspect what we have been lacking or what more is to be done to be the winner in your chosen arena of work or ambition.

Very relatable! 👍

**Comment by swapagarwal** [comment]
Inspiring story @PragatiVerma18! 👏
Can't wait to see this at https://thenextbigwriter.tech/

File: my-tech-journey/genesis.md (hunk 3 above)
Comment (PragatiVerma18) [reply my-tech-journey/genesis.md:12]: Okay, I will add a few more points on that. Thanks for suggestions. 😀

File: my-tech-journey/genesis.md (hunk 4 above)
Comment (PragatiVerma18) [reply my-tech-journey/genesis.md:18]: Thanks 😃

**Comment by PragatiVerma18** [comment]
> Inspiring story @PragatiVerma18! 👏
> Can't wait to see this at https://thenextbigwriter.tech/

Thanks for your valuable suggestions and feedback. Should I publish it?

**Comment by iamshadmirza** [comment]
> > Inspiring story @PragatiVerma18! 👏
> > Can't wait to see this at https://thenextbigwriter.tech/
> 
> Thanks for your valuable suggestions and feedback. Should I publish it?

Hi Pragati, awesome write up. Can you please put the folder inside the general directory?  
We are categorizing articles into folders, I will quickly merge the PR then you can submit it to hashnode.  
Select "The Next Big Writer" under publication when you are submitting and it will be published.

**Comment by PragatiVerma18** [comment]
Please merge [this one](https://github.com/iamshadmirza/TheNextBigWriter/pull/23) and close this one. @iamshadmirza @swapagarwal

**Comment by PragatiVerma18** [comment]
Already merged. Thanks.

**Pull request closed without merging**